spec/decorators/user_decorator_spec.rb

```diff
@@ -214,7 +214,9 @@
     end
 
     context 'second factor locked out a while ago' do
-      let(:locked_at) { Time.zone.now - UserDecorator::DEFAULT_LOCKOUT_PERIOD - 1.second }
+      let(:locked_at) do
+        Time.zone.now - IdentityConfig.store.lockout_period_in_minutes.minutes - 1.second
```

Comment:
could use more rails activesupport magic!

```suggestion
        IdentityConfig.store.lockout_period_in_minutes.minutes.ago - 1.second
```